package.json

```diff
@@ -15,13 +15,12 @@
   },
   "author": "",
   "license": "MIT",
-  "dependencies": {
+  "dependencies": {},
```

Comment:
intended ?

Reply:
yes, published pacakge dont need any deps, and user dont need to install them